tasks/custom_task/custom_task.py

```diff
@@ -0,0 +1,81 @@
+"""
+This module defines the InvokeLogger context manager to log the invoke task information to the DD_INVOKE_LOGS_PATH.
+This will then be uploaded to Datadog's backend with a correct Log Agent configuration.
+"""
+
+from time import perf_counter
+from getpass import getuser
+from datetime import datetime
+import logging
+import traceback
+from invoke import Context
+
+DD_INVOKE_LOGS_PATH = "/tmp/dd_invoke.log"
+
+
+def log_invoke_task(name: str, module: str, task_datetime: str, duration: float, task_result: str) -> None:
+    """
+    Logs the task information to the DD_INVOKE_LOGS_PATH file.
+    This should be uploaded to Datadog's backend with a correct Log Agent configuration:
+    ```
+    logs:
+    - type: file
+        path: "/tmp/dd_invoke.log"
+        service: "dd_invoke_logs"
+        source: "invoke"
+    ```
+    """
+    logging.basicConfig(filename=DD_INVOKE_LOGS_PATH, level=logging.INFO, format='%(message)s')
```

Comment:
Could you do this when you init the logger?

Reply:
WDYM ? This is where I init the logger, isn't it?

Reply:
Discussed offline: fixed with [47d6590](https://github.com/DataDog/datadog-agent/pull/25147/commits/47d6590c7ef024390233d04ef38d048c2f9f5b6a).